readBoundaries.R

```diff
@@ -0,0 +1,27 @@
+library(sf)
+library(ggplot2)
+library(dplyr)
+library(lubridate)
+library(tidyverse)
```

Comment:
Again, if you intend to source this file in `app.R`, try not to include libraries unnecessary to this file.
For example `readBoundaries.R` does not use ggplot2 or lubridate